docs/4.2/kubernetes_ssh.md

```diff
@@ -125,7 +125,7 @@ configured. In this guide we'll look at two common configurations:
 * **Open source, Teleport Community edition** configured to authenticate users via [Github](admin-guide.md#github-oauth-20).
   In this case, we'll need to map Github teams to Kubernetes groups.
 
-* **Commercial, Teleport Enterprise edition** configured to authenticate users via [Okta SSO](ssh_okta.md).
+* **Commercial, Teleport Enterprise edition** configured to authenticate users via [Okta SSO](enterprise/sso/ssh_okta.md).
```

Comment:
I'm not sure why we use "Commercial" here as we don't refer to Teleport Enterprise this way anywhere else.

Reply:
- Open source, Teleport Community edition
- Commercial, Teleport Enterprise edition

^ I think this makes a lot of sense and matches other OSS projects, but I when referring to the feature short team I'll call it Commercial, Teleport Enterprise edition